src/include/ndpi_protocol_ids.h

```diff
@@ -301,4 +303,4 @@ typedef enum {
 #define NDPI_PROTOCOL_NO_MASTER_PROTO    NDPI_PROTOCOL_UNKNOWN
 #define NDPI_MAX_SUPPORTED_PROTOCOLS     NDPI_LAST_IMPLEMENTED_PROTOCOL
 #define NDPI_MAX_NUM_CUSTOM_PROTOCOLS    (NDPI_NUM_BITS-NDPI_LAST_IMPLEMENTED_PROTOCOL)
-#endif
+#endif
```

Comment:
nit: There is no newline at the end of file (other files in the repo all have one). I don't know how strict the maintainers are about this, but I personally would fix it.

Reply:
thank you i fix it